USER_GUIDE.md

```diff
@@ -371,13 +373,17 @@ query = {
 response = client.plugins.alerting.acknowledge_alert(query)
 print(response)
 ```
-## Using IAM credentials for authentication
+## Using different authentication schemas
+
+It is possible to use different schemas for the authentication to OpenSearch. The parameters of `connection_class` and `http_auth` can be used for this. As an example, here we have how to authenticate using IAM credentials and Kerberos authentication.
```

Comment:
schemas -> methods

As an example, here we have -> The following example shows how to authenticate using IAM credentials via Kerberos.